test/truefi2/TrueLender2.test.ts

```diff
@@ -310,13 +399,42 @@ describe('TrueLender2', () => {
         await expect(lender.connect(borrower).fund(loan1.address))
           .to.be.revertedWith('TrueLender: Loan risk is too high')
       })
+
+      it('loan term exceeds max term', async () => {
+        await rater.connect(borrower).submit(loan1.address)
+        await rater.yes(loan1.address)
+        await timeTravel(7 * DAY + 1)
+        await lender.setLongLoanMinTerm(DAY * 364)
+        await lender.setMaxLoanTerm(DAY * 364)
```

Comment:
Why 364 not 365?

Reply:
Because loan1 has a 1 year term and we want max term to be lower for this test

Reply:
I think it would be more obvious if it was `DAY * 365 - 1`